crates/processing_render/src/image.rs

```diff
@@ -140,10 +140,16 @@ pub fn from_handle(
 }
 
 pub fn load(In(path): In<PathBuf>, world: &mut World) -> Result<Entity> {
-    #[cfg(feature = "python")]
-    let path = AssetPath::from_path_buf(path).with_source(AssetSourceId::from("assets_directory"));
+    let config = world.resource_mut::<Config>();
+    let path: AssetPath = match config.get(ConfigKey::AssetRootPath) {
+        Some(_) => {
+            AssetPath::from_path_buf(path).with_source(AssetSourceId::from("assets_directory"))
```

Comment:
This name (`"assets_directory"`) might be confusing for contributors coming from bevy (as that's canonically `assets/`) but I can't think of a better name off the top of my head.